src/main/kotlin/org/rust/ide/annotator/RsErrorAnnotator.kt

```diff
@@ -244,22 +244,22 @@ class RsErrorAnnotator : Annotator, HighlightRangeExtension {
     private fun checkImpl(holder: AnnotationHolder, impl: RsImplItem) {
         val trait = impl.traitRef?.trait ?: return
         val traitName = trait.name ?: return
-
-        val canImplement = trait.functionList.associateBy { it.name }
-        val mustImplement = canImplement.filterValues { it.isAbstract }
+        
```

Comment:
I think the name can be obtained from `traitRef.text`. Or we could only resolve the trait if we really found an error and need to emit the annotation. But not always.